vendor/gopkg.in/yaml.v2/apic.go

```diff
@@ -52,6 +52,11 @@ func yaml_reader_read_handler(parser *yaml_parser_t, buffer []byte) (n int, err
 	return parser.input_reader.Read(buffer)
 }
 
+// Set a max depth.
+func yaml_parser_set_max_flow_level(parser *yaml_parser_t, max_flow_level int) {
```

Comment:
It seems that the functions in this file are rewrites of https://github.com/yaml/libyaml/blob/master/src/api.c, where there is no yaml_parser_set_max_flow_level. Shall we define this new function elsewhere?